support coco stuff-10k/164k (#625)
open-mmlab/mmsegmentation: docs/dataset_prepare.md

python tools/convert_datasets/pascal_context.py data/VOCdevkit data/VOCdevkit/VOC2010/trainval_merged.json
```

### COCO Stuff 10k

The data could be downloaded [here](http://calvin.inf.ed.ac.uk/wp-content/uploads/data/cocostuffdataset/cocostuff-10k-v1.1.zip) by wget.

> If you would like to use COCO Stuff 10k dataset, please run following command to convert augmentation annotations into proper format.

Reviewer: ```suggestion
For  COCO Stuff 10k dataset, please run the following commands to download and convert the dataset.
```

```shell
# download
cd data
mkdir coco_stuff10k && cd coco_stuff10k
wget http://calvin.inf.ed.ac.uk/wp-content/uploads/data/cocostuffdataset/cocostuff-10k-v1.1.zip

# unzip
unzip cocostuff-10k-v1.1.zip
cd ../..

# --nproc means 8 process for conversion, which could be omitted as well.
python tools/convert_datasets/coco_stuff10k.py data/coco_stuff10k --nproc 8
```

By conversion, masks in `data/coco_stuff10k/masks/*.png` are used for COCO Stuff 10k training and testing.

### COCO Stuff 164k

> The data could be downloaded and converted augmentation annotations into proper format. by following command.

Reviewer: ```suggestion
For  COCO Stuff 164k dataset, please run the following commands to download and convert the augmented dataset.
```
